File: MPI/mpicheck/tests/functions/4.0/MPI_Session_get_info.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
        call mpi_session_get_info(var_0, var_1, var_2)
       call pmpi_session_get_info(var_0, var_1, var_2)
        end program main
    

File: MPI/mpicheck/tests/functions/4.0/MPI_Session_get_info.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Session var_0;
    MPI_Info *var_1;
    int ret;
    /* calls */
    ret = MPI_Session_get_info(var_0, var_1);
    ret = PMPI_Session_get_info(var_0, var_1);
    return 0;
}
